src/figuresClasses.ts

```diff
@@ -1,19 +1,111 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
```

Comment:
Try not to use magic values, for this string variables can be expressed as enam.
https://kyleshevlin.com/what-are-magic-values